src/PyCall.jl

```diff
@@ -708,6 +708,14 @@ for (mime, method) in ((MIME"text/html", "_repr_html_"),
 end
 
 #########################################################################
+# Expose Python docstrings to Julia doc system
+
+if isdefined(Docs, :getdoc)  # Introduced in Julia .6
+    Docs.getdoc(o::PyObject) = Markdown.parse(o[:__doc__])
```

Comment:
The main advantage of returning `MD(Code(string))` is probably for the case where there are multiple docstrings, some of which might be markdown, since there is a `Base.Docs.catdoc(::MD...)`` method.  But since a `PyObject` will presumably only have a single docstring, `Text` seems like it should be fine.